sdk/python/jobs/automl-standalone-jobs/automl-forecasting-recipes-univariate/automl-forecasting-recipe-univariate-run.ipynb

```diff
@@ -0,0 +1,1315 @@
+    "- Create an `AutoML time-series forecasting Job` with the 'forecasting()' factory-fuction\n",
+    "- Train the model using AmlCompute by submitting/running the AutoML forecasting training job\n",
+    "- Obtain the model and use it to generate forecast\n",
+    "\n",
+    "**Motivations** - This notebook explains how to setup and run an AutoML forecasting job. This is one of the nine ML-tasks supported by AutoML. Other ML-tasks are 'regression', 'classification', 'image classification', 'image object detection', 'nlp text classification', etc.\n",
+    "\n",
+    "In this example we use the bike-sharing service dataset to showcase how you can use AutoML for a simple forecasting problem and explore the results."
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## 1. Connect to Azure Machine Learning Workspace\n",
+    "\n",
+    "The [workspace](https://docs.microsoft.com/en-us/azure/machine-learning/concept-workspace) is the top-level resource for Azure Machine Learning, providing a centralized place to work with all the artifacts you create when you use Azure Machine Learning. In this section we will connect to the workspace in which the job will be run."
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "### 1.1. Import the required libraries"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "source": [
+    "# Import required libraries\n",
+    "from azure.identity import DefaultAzureCredential\n",
+    "from azure.ai.ml import MLClient\n",
+    "from azure.ai.ml.constants import AssetTypes\n",
+    "from azure.ai.ml import automl\n",
+    "from azure.ai.ml import Input\n",
+    "\n",
+    "import json\n",
+    "import pandas as pd\n",
+    "import os\n",
+    "import shutil\n",
+    "import yaml"
+   ],
+   "outputs": [],
+   "execution_count": null,
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "### 1.2. Configure workspace details and get a handle to the workspace\n",
+    "\n",
+    "To connect to a workspace, we need identifier parameters - a subscription, resource group and workspace name. We will use these details in the `MLClient` from `azure.ai.ml` to get a handle to the required Azure Machine Learning workspace. We use the default [default azure authentication](https://docs.microsoft.com/en-us/python/api/azure-identity/azure.identity.defaultazurecredential?view=azure-python) for this tutorial. Check the [configuration notebook](../../configuration.ipynb) for more details on how to configure credentials and connect to a workspace."
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "#### Show Azure ML Workspace information"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "source": [
+    "credential = DefaultAzureCredential()\n",
+    "ml_client = None\n",
+    "# import pdb;pdb.set_trace()\n",
```

Comment:
Dead code. Please remove it.

Reply:
Please explain more here.

Reply:
The comment is to remove the dead code (line 91). Is there a reason to keep it?

Reply:
Oh, I have removed the line. Thank you.